packages/shared/src/lib/geolocation.ts

```diff
@@ -545,6 +547,17 @@ const extendedCurrencySymbol = {
     International: '$',
 };
 
+export const isRegion = (val: string): boolean => {
+    return Object.keys(countryGroups).includes(val);
```

Comment:
I think we're just looking up a key in an object here? Would it be simpler to just index into `countryGroups` like this:

```suggestion
    return !!countryGroups[val];
```

I'm not too worried about performance, but it feels more intuitive to use an object like this, rather than convert the keys to an array and have to iterate over those using `includes`. What do you think?

Reply:
Makes good sense. Will update accordingly.